vscode/src/debugger/output.ts

```diff
@@ -7,7 +7,7 @@ export function createDebugConsoleEventTarget(out: (message: string) => void) {
   const eventTarget = new QscEventTarget(false);
 
   eventTarget.addEventListener("Message", (evt) => {
-    out(`Message: ${evt.detail}`);
+    out(`${evt.detail}`);
```

Comment:
There's no point in having an interpolated string any more. May as well just `out(evt.detail);`.